repo/blob/filesystem/filesystem_storage.go

```diff
@@ -306,7 +308,7 @@ func (fs *fsStorage) TouchBlob(ctx context.Context, blobID blob.ID, threshold ti
 
 		//nolint:wrapcheck
 		return osi.Chtimes(path, n, n)
-	}, fs.Impl.(*fsImpl).isRetriable) //nolint:forcetypeassert
+	}, fsimp.isRetriable)
```

Comment:
Forgive my ignorance, what is the relevance of this change here?

Reply:
the assertion doesn't do anything - so I removed it.